content/docs/use-cases/data-registries.md

```diff
@@ -164,9 +164,9 @@ the project dependency metadata in the import stage (DVC-file).
 
 ### Programmatic reusability of DVC data
 
-Our Python API, included with the `dvc` package installed with DVC, includes the
-`open` function to load/stream data directly from external <abbr>DVC
-projects</abbr>:
+Our Python API (`dvc.api`), included with the `dvc` package installed with DVC,
```

Comment:
Here I would actually prefer to link `[Python API]` manually for readability.

Reply:
Done in https://github.com/iterative/dvc.org/pull/1283/commits/036672de4d6b2253a6462a2b152c251f6d0f03d3 along with a couple more instances I found.